tests/integration_tests/data_steward/retraction/retract_deactivated_pids_test.py

```diff
@@ -0,0 +1,182 @@
+        print('**************************************************************')
+
+    def setUp(self):
+        self.hpo_id = 'fake'
+        self.project_id = app_identity.get_application_id()
+        self.fake_project_id = 'fake-project-id'
+        self.bq_dataset_id = bq_utils.get_dataset_id()
+        self.ticket_number = 'DCXXX'
+        self.pid_table_id = 'pid_table'
+        self.deactivated_ehr_participants = [(1, '2007-01-01'),
+                                             (2, '2008-02-01'),
+                                             (1234567, '2019-01-01')]
+
+    @mock.patch(
+        'retraction.retract_deactivated_pids.get_date_info_for_pids_tables')
+    @mock.patch('bq_utils.list_datasets')
+    def test_integration_queries_to_retract_from_fake_dataset(
+        self, mock_list_datasets, mock_retraction_info):
+        mock_list_datasets.return_value = [self.bq_dataset_id]
+        d = {
+            'project_id': [
+                self.project_id, self.project_id, self.project_id,
+                self.project_id, self.project_id, self.project_id
+            ],
+            'dataset_id': [
+                self.bq_dataset_id, self.bq_dataset_id, self.bq_dataset_id,
+                self.bq_dataset_id, self.bq_dataset_id, self.bq_dataset_id
+            ],
+            'table': [
+                'fake_condition_occurrence', 'fake_drug_exposure',
+                'fake_measurement', 'fake_observation',
+                'fake_procedure_occurrence', 'fake_visit_occurrence'
+            ],
+            'date_column': [
+                'condition_end_date', 'drug_exposure_end_date',
+                'measurement_date', 'observation_date', 'procedure_date',
+                'visit_end_date'
+            ]
+        }
+        retraction_info = pd.DataFrame(data=d)
+        mock_retraction_info.return_value = retraction_info
+
+        # Create and load person_ids and deactivated_date to pid table
+        bq_utils.create_table(self.pid_table_id,
+                              retract_deactivated_pids.PID_TABLE_FIELDS,
+                              drop_existing=True,
+                              dataset_id=self.bq_dataset_id)
+        bq_formatted_insert_values = ', '.join([
+            '(%s, "%s")' % (person_id, deactivated_date)
+            for (person_id,
+                 deactivated_date) in self.deactivated_ehr_participants
+        ])
+        q = INSERT_PID_TABLE.format(
+            dataset_id=self.bq_dataset_id,
+            pid_table_id=self.pid_table_id,
+            person_research_ids=bq_formatted_insert_values)
+        bq_utils.query(q)
+
+        job_ids = []
+        row_count_queries = []
+        hpo_table_list = []
+
+        # Load the cdm files into dataset
+        for cdm_file in test_util.NYC_FIVE_PERSONS_FILES:
+            cdm_file_name = os.path.basename(cdm_file)
+            cdm_table = cdm_file_name.split('.')[0]
+            hpo_table = bq_utils.get_table_id(self.hpo_id, cdm_table)
```

Comment:
same as above.  should use bigquery client for new work.

Reply:
this function is not in `utils.bq` let me know if I am ok to add? It is just a string concatenation, so I could also add the following.
`hpo_table = self.hpo_id + '_' + cdm_table`

`bq_utils.get_table_id`:
```
def get_table_id(hpo_id, table_name):
    """
    Get the bigquery table id associated with an HPOs CDM table
    :param hpo_id: ID of the HPO
    :param table_name: name of the CDM table
    :return: the table id
    """
    if hpo_id is None:
        return table_name
    # TODO revisit this; currently prefixing table names with hpo_id
    return hpo_id + '_' + table_name
```

Reply:
This might be something to bring up at the next eng_sync meeting.  For now, just leave this line unchanged.  I'm not sure if we should move those into `utils.bq` or somewhere else.

Reply:
sounds good, I'll make sure to add to the agenda for the next eng sync.